webapp/redux/middleware/middleware.js

```diff
@@ -342,14 +342,14 @@ export default (store) => (next) => (action) => {
     case CREATE_NETWORK: {
       next(GeppettoActions.waitData('Instantiating the NetPyNE Model', GeppettoActions.layoutActions.SET_WIDGETS));
 
-      checkParametersThen(() => instantiateNetwork({}))
+      checkParametersThen(() => instantiateNetwork({}), false, "Creating the NetPyNE Model")
```

Comment:
I would use the same message we had in the line above -> "Instantiating the NetPyNE Model"